test/storage/sequelize.test.ts

```diff
@@ -340,6 +359,16 @@ describe('sequelize', () => {
 	});
 
 	describe('executed', () => {
+		it('validates migration types', async () => {
+			const storage = new Storage({ sequelize: helper.sequelize });
+
+			jest.spyOn(storage.model, 'findAll').mockResolvedValueOnce([{ name: 123 } as any]);
```

Comment:
Question: I noticed in the code for the `"lazy-initializes model"` test, there are calls to `.mockClear()` the spies, while this test does not have it. Why is it different here?

Reply:
No good reason. To be hones the `.mockClear()` in the other places doesn't really do anything, since nobody else is using the spy. And even if they were, the usual way to ensure tests don't pollute each other is to call `jest.clearAllMocks()` in a `beforeEach`. So I'll remove the `mockClear`s from the other test.